pkg/services/rpcsrv/subscription_test.go

```diff
@@ -261,6 +268,39 @@ func TestFilteredSubscriptions(t *testing.T) {
 				require.Equal(t, "my_pretty_notification", n)
 			},
 		},
+		"notification matching contract hash and parameter": {
+			params: `["notification_from_execution", {"contract":"` + testContractHash + `", "parameters":[{"type":"Any","value":null},{"type":"Hash160","value":"449fe8fbd4523072f5e3a4dfa17a494c119d4c08"}]}]`,
+			check: func(t *testing.T, resp *neorpc.Notification) {
+				rmap := resp.Payload[0].(map[string]any)
+				require.Equal(t, neorpc.NotificationEventID, resp.Event)
+				c := rmap["contract"].(string)
+				require.Equal(t, "0x"+testContractHash, c)
+				// it should be exact unique "Init" call sending all the tokens to the contract itself
+				parameters := rmap["state"].(map[string]any)["value"].([]any)
+				require.Len(t, parameters, 3)
+				// sender
+				transferReceiverType := parameters[1].(map[string]any)["type"].(string)
+				require.Equal(t, smartcontract.Hash160Type.ConvertToStackitemType().String(), transferReceiverType)
+				transferReceiver := parameters[1].(map[string]any)["value"].(string)
```

Comment:
s/transferReceiverType/toType
s/transferReceiver/to

Because it's commonly used `[from, to, amount, args]` cortege.